client/app/instructor/dashboard/dashboard.controller.js

```diff
@@ -9,10 +9,27 @@ angular.module('venueApp')
 
     Auth.getCurrentUser((user) => {
       $scope.user = user;
-      User.get({id: user._id, withSections:true, withEvents: true, withSectionsCourse:true}, (usr) => {
+      User.get({id: user._id, withSections:true withEvents: true, withSectionsCourse:true}, (usr) => {
```

Comment:
You don't need to send the id, or even call getCurrentUser with Auth for the logged in user.